### PR TITLE
2022_02_20_upgrade_pkt_chat_software

### PR DESCRIPTION
**Pull request by thierrymarianne**
Dear NS Committee,

After attending recent [live community event](https://odysee.com/@cjdns:0/Live-with-Caleb-James-DeLisle:1), 
I'd like to make this proposal official with regards to pkt.chat underlying software ([Matterfoss](https://github.com/cjdelisle/Matterfoss)).

Thank you for your guidance about how I could make it better,
and your comments regarding this project feasibility.

Thierry

### CODE REVIEW

**Comment by cjdelisle** [comment]
Hello @thierrymarianne , thanks for making this proposal.

This is not an approved communication of the NS, this is just my personal comments. Though I can't guarantee anything, and in fact I will abstain from voting on this project myself, I think it is excellent, my only comment is that perhaps you would want to re-price the individual deliverables in a more stable currency such that if PKT price goes down, you will be protected and if PKT price goes up, the community will be.

The NS requires that the maximum amount of PKT for the project be specified, in case of a major decrease in the price of PKT, the project would be accepted partially delivered and the NS would consider a follow-on proposal to complete the work.

Perhaps take a look a this template: https://cryptpad.fr/code/#/2/code/view/lHORne-EdOqRVL2kBQ7SCPpt8U9SoRF5utOCFjnaGVs/ which shows the process of delivering a milestone with the price of PKT at that time.

**Comment by thierrymarianne** [comment]
Thank you for your valuable feedback. 
I've made the requested changes and fixed some inconsistencies.

I look forward to learning more from the committee regarding this proposal.

File: projects/2022_02_20-pkt.chat_version_upgrade.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+ 10. (Optional) Offer support to project maintainer up to half a week worth of post-release maintenance (hourly billed)
+
+### Milestone 1 (Kickoff)
+
+Upon agreement during the kickoff meeting regarding tagged versions to be targeted for integration from upstream,  
+1/10 of the total project PKT contribution (27.5K PKT) will be provided. 
+
+ * The cost of this milestone is 467.5 EUR and  
+ I declare that the current trading price is 0.017 EUR per PKT on the 20th of February 2020
+ * Therefore I will request a transfer of 27,500 PKT
```
[the remark this answers is not in the document]
Comment (thierrymarianne) [reply projects/2022_02_20-pkt.chat_version_upgrade.md:98]: I was off by two years, it was supposed to be the date when this PR was submitted (I intended to make this number correct according to price.pkt.com though).

I'll replace these definitely **wrong** price-dates pair with `to be determined on milestone submission` if that is better suited.

**Comment by cjdelisle** [comment]
It looks like the objective is achieved following your edits.

**Comment by cjdelisle** [comment]
Typically an audit certificate would be something for a large project where it's important to be sure of where the money went. You did not check the box promising to provide an audit certificate so it is not relevant to this project.

**Comment by thierrymarianne** [comment]
Duly noted. Thanks!

**Comment by cjdelisle** [comment]
Congratulations, this project was selected for funding.

In order to optimize future project proposals it is recommended to look at the notes from the reviewers: https://github.com/pkt-cash/ns-projects/blob/master/vote_2022_02_10.py

Wishing you an excellent project ahead, and excited to stop seeing the warning in the pkt.chat app :)

**Comment by thierrymarianne** [comment]
Amazing! Thank you! 🥳 
Another PR has been opened to discuss [Kick-off scheduling and related payment request](https://github.com/pkt-cash/ns-projects/pull/110)

**Pull request merged**